xml/vt_qemu_tpm.xml

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="tpm">
+ <title>Software TPM emulator</title>
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+  </dm:docmanager>
+ </info>
+ <sect1 xml:id="tpm-introduction">
+  <title>Introduction</title>
+  <para>
+   Trusted Platform Module (TPM) is a cryptoprocessor that secures hardware by using cryptographic keys.
+   For developers who use TMP to develop security features, a software TPM emulator is a convenient solution.
+   Compared to a hardware TPM device, the emulator has no limit for the number of guests that can access it.
+   Also, it is simple to switch between versions TPM 1.2 and TPM 2.0.
+   &qemu; supports the software TPM emulator that is included in the <package>swtpm</package> package.
+  </para>
+ </sect1>
+ <sect1 xml:id="tpm-prerequisite">
+  <title>Prerequisites</title>
+  <para>
+   Before you can install and use the software TPM emulator, you need to install the &libvirt; virtualization environment.
+   Refer to <xref linkend="vt-installation-yast"/> and install one of the provided virtualization solutions.
+  </para>
+ </sect1>
+ <sect1 xml:id="tpm-installation">
+  <title>Installation</title>
+  <para>
+   To use the software TPM emulator, install the <package>swtpm</package> package:
+  </para>
+<screen>&prompt.sudo;zypper install swtpm</screen>
+ </sect1>
+ <sect1 xml:id="tpm-qemu">
+  <title>Using swtpm with &qemu;</title>
```

Comment:
Should we use a [command] tag around this? I see from the next paragraph that it **is** a command.